agents/windows/plugins/win_license.bat

```diff
@@ -5,4 +5,13 @@ REM * plugin to gather and output Windows activation status
 REM ***
 
 echo ^<^<^<win_license^>^>^>
-cscript //NoLogo %windir%/System32/slmgr.vbs /dli
+if not exist "%temp%\slmgr.vbs" (
+    copy "%windir%\system32\slmgr.vbs" "%temp%\slmgr.vbs" > NUL
+)
```

Comment:
I would rather overwrite the file if it exists. Given that we delete the file after execution, it would be weird to run something that already exists.